src/backend/efiling-api/jag-efiling-api.yaml

```diff
@@ -223,14 +223,14 @@ paths:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/UserDetails"
+              $ref: "#/components/schemas/CreateCsoAccountRequest"
       responses:
         "201":
           description: Account Created
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/UserDetails"
+                $ref: "#/components/schemas/UserFullDetails"
```

Comment:
question: why was this changed to `userfulldetails`? how come `userdetails` wasnt a good name for the model?

Reply:
userdetails is used as well, the full details includes first name last name ...